### DISCUSSION TITLE
[예인] answer 패키지 하위 패키지 추가하였어요.

### DISCUSSION THREAD

**Discussion by brightyein** [KKLP01/study#11] [Show and tell]
answer 패키지 안에 각 팀원별 정답 적는 패키지를와 클래스를 추가하였어요.

Yujinanswer, YeinAnswer 와 같은 클래스를 최종 정답을 실행하는 메인클래스로 두어주세요.
(각 문제에 따라 추가적으로 발생하는 클래스들은 자유롭게 자신의 폴더에 생성하셔도 됩니다)

문제 푸시기 전 main 클래스를 pull 하신 뒤 시작하시기 바랍니당!
감사합니다. 화이팅!

**Comment by brightyein** [comment]
아! 그리고 문제 다 출제하셨으니, 이슈에 아래의 컨벤션에 따라 '문제 풀기' 이슈생성 부탁드려요:)

- 예시 -
- 
[이름] 5월 1주 문제풀기
- [ ] 예인
- [ ] 도연
- [ ] 유진

**Reply by brightyein** [reply]
그냥 템플릿으로 만들어 두었습니다~! (문제내기/ 문제풀기)
스터디 날짜에 해당하는 마일스톤 추가해주시고, 내용은 본인에 맞게 커스텀하셔요~~

**Reply by kimdoyeon304** [reply]
우와 감사합니다!
깃헙 굉장히 잘 활용하시는거 같아요~
